src/modules/layouts/blogPost_layout.js

```diff
@@ -37,20 +74,25 @@ export default ({children, pageContext}) => {
 					<BlogAuthor author={authors} isContributors/>
 				</div>
 			}
-			{recommendations &&
+			{(otherPosts && otherPosts.length > 0) &&
```

Comment:
I didn't touch the design here yet, I will wait for this pull request to be merged and solve conflicts with mine and adjust this design.